engine/include/cubos/engine/splitscreen/plugin.hpp

```diff
@@ -0,0 +1,21 @@
+/// @dir
+/// @brief @ref splitscreen-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point
+/// @ingroup splitscreen-plugin
+
+#pragma once
+#include <cubos/engine/cubos.hpp>
```

Comment:
```suggestion
#pragma once

#include <cubos/engine/cubos.hpp>
```

Some space to breathe :relieved: